README.md

```diff
@@ -1 +1,80 @@
-# Capstone-project
\ No newline at end of file
+# Capstone-project
+
+#### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./MIT.md) of the design.
+<br />
+
+# LEADWAY COMMONS GLOBAL SUMMIT
+
+> This project is about leadway common summit which bring agricultural expertise around the wotld together for a conference. It was founded in 2017 , the program was held in Nigeria in the year 2022.
+
+
+This project is a part of my Microverse curriculum. This is made as Final Project (Capstone), of 1st Module at Microverse in order to attract job opportunities. The main objective is to develop a professional website which showcase my professional skills and also will meet the requirements of a client and be able to meet deadlines.
+
+## Live
+Go live with me using this link https://www.loom.com/share/aafc27401a3d49e886d5f814fa74d449
```

Comment:
- [OPTIONAL] Kindly you can put the Go live with me using this link in the `[`] and the loom url in the `()` then you will have a resul like [Go live with me using this link](https://www.loom.com/share/aafc27401a3d49e886d5f814fa74d449) the link will be in the description. it looks nice

Reply:
Thanks I have rectify that